src/scripts/main.js

```diff
@@ -1,3 +1,69 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const appendColumn = document.querySelector('.append-column');
+const removeRow = document.querySelector('.remove-row');
+const removeColumn = document.querySelector('.remove-column');
+const tabl = document.getElementsByTagName('tbody')[0];
+let amountColum = 4;
+let amountRow = 4;
+
+const AddRow = () => {
+  const tr = document.createElement('tr');
+
+  if (amountRow >= 10) {
+    return;
+  }
+
+  for (let i = 0; i < amountColum; i++) {
+    const td = document.createElement('td');
+
+    tr.append(td);
+  }
+  tabl.prepend(tr);
```

Comment:
here you can add the row without for loop. you can just copy the first row and add it to the table
```suggestion
  tabl.append(tabl.firstElementChild.cloneNode(true));
```